mercadopago/resources/order.py

```diff
@@ -33,3 +32,131 @@ def create(self, order_object, request_options=None):
             raise ValueError("Param order_object must be a Dictionary")
 
         return self._post(uri="/v1/orders", data=order_object, request_options=request_options)
+
+    def get(self, order_id, request_options=None):
+        """[Click here for more info](https://www.mercadopago.com.br/developers/en/reference/order/online-payments/get-order/get ) # pylint: disable=line-too-long
+
+        Args:
+            order_id (str): The Order ID
+            request_options (mercadopago.config.request_options, optional): An instance of
+            RequestOptions can be pass changing or adding custom options to ur REST call.
```

Comment:
[req] Erro ortográfico
```suggestion
            RequestOptions can be pass changing or adding custom options to the REST call.
```